proto/confio/poe/v1beta1/genesis.proto

```diff
@@ -87,6 +87,28 @@ message SeedContracts {
 
   ArbiterPoolContractConfig arbiter_pool_contract_config = 13
       [ (gogoproto.jsontag) = "arbiter_pool_contract_config,omitempty" ];
+
+  MixerContractConfig mixer_contract_config = 14
+      [ (gogoproto.jsontag) = "mixer_contract_config,omitempty" ];
+}
+
+// Mixer contract setup. See
+// https://github.com/confio/poe-contracts/tree/main/contracts/tg4-mixer
+message MixerContractConfig {
+  message Sigmoid {
+    uint64 max_rewards = 1;
```

Comment:
Note here and others: this is going to be renamed to `max_points` for PoE contracts v0.11.0